### ISSUE TITLE
Join mailing list form on uf website

### ISSUE THREAD

**Issue by SarahBaghdadi** [userfrosting/web#17]
Perhaps located in the footer, or on the homepage as a section.

**Comment by alexweissman** [comment]
Yup.  I'd like to store email addresses in a DB, but Grav is not DB-based.  So, we will have to figure out a good way to integrate the two.  Maybe we can host a simple API on a subdomain and have just the form on Grav, post to our API.

**Comment by JoshNorris** [comment]
Why not use a service such as MailChimp? Sure, it's not the way of building it yourself, but it should be mostly quick and easy to setup and get it working as a middle ground till you figure out a better way to do it.

**Comment by alexweissman** [comment]
hmmmm...

**Comment by SarahBaghdadi** [comment]
At this point we have a list from people who sign up for the demo site, and its storing their email preferences. So I think we would need to link up our form with that database, is that correct?

**Comment by alexweissman** [comment]
you mean, so that people can sign up for the mailing list without needing to create an account on the demo server?

**Comment by SarahBaghdadi** [comment]
Yeah. I mean I guess most people will demo. But I think its still a good practice to have some active way of signing up, and you might catch more people that way.

**Comment by alexweissman** [comment]
Ok that's a good point, but I'd have to think about how to implement this.  Maintaining two separate lists/tables would be a pain.
